tests/test_envattrs.py

```diff
@@ -23,6 +23,24 @@ class Foo:
         envattrs.load(Foo, "BAR", {"FOO_BAR": "baz"})
 
 
+def test_load_with_leading_underscore():
+    @attr.s
+    class Foo:
+        _bar = attr.ib()
+
+    instance = envattrs.load(Foo, "FOO", {"FOO_BAR": "baz"})
+    assert instance._bar == "baz"
+
+
+def test_load_with_non_leading_underscore():
```

Comment:
maybe add one with a trailing underscore just to cover all our bases?

Reply:
done. All your base are belong to us